lib/ui/validation/form.js

```diff
@@ -52,6 +52,9 @@
       if(id && this.violations[id]) {
         delete this.violations[id];
       }
+      if(_.isEmpty(this.violations)) {
+        this.ngForm.$setValidity('av', _.isEmpty(this.violations));
```

Comment:
you could just set it to true at this point instead of re-running the `isEmpty` check since it will have to truthy in order to hit this point.